### ISSUE TITLE
Adding Batch Normalization Examples

### ISSUE THREAD

**Issue by kkweon** [hunkim/DeepLearningZeroToAll#75]
## Abstract
TL;DR Let's add a batch norm
## Issue
The same issue has been called out a few days ago, but I don't see a PR coming up.
Anyway, I think it's a good idea to add batch normalization examples because it's quite confusing for beginners.

Why it's confusing: because when there is a batch normalization layer, exponential moving average (momentum) must be calculated at training time. However, this is not included in the computation graph because it's not necessary to get the output of a network.

Therefore, one must include 
```python
update_ops = tf.get_collection(tf.GraphKeys.UPDATE_OPS, scope=name)
with tf.control_dependencies(update_ops):
    train_op = optimizer(lr).minimize(self.loss)
```

There are other ways but this is the simplest one.

## Suggestion
* I am thinking of adding [this Jupyter notebook](https://nbviewer.jupyter.org/gist/kkweon/e73aad30e3dd096bca7a50d126749518)
* Simpler example may work better

**Comment by hunkim** [comment]
Sounds really good. Perhaps, lab-10-6-mnist_batch_norm?

**Comment by kkweon** [comment]
@hunkim Do you think the [Jupyter notebook link](https://nbviewer.jupyter.org/gist/kkweon/e73aad30e3dd096bca7a50d126749518) is easy to understand?

If so, I'm going to add comments to make it clear, but I'm going to keep the codes as it is.

**Comment by hunkim** [comment]
@kkweon Yes. I think it's the way to go.
